tests/Session/test_profile.py

```diff
@@ -57,12 +59,13 @@ def testListFoodStorages(self):
         self.assertEqual([freezer_name, fridge_name], self.profile.listFoodStorages(FOOD_STORAGE)) 
 
     def testAddFoods(self):
-        # Check self.profile.addFoods returns 0 = tests that food storage is not open yet 
+        # Should return 0 if food storage is not open yet.
+        self.assertEqual(0, self.profile.addFoods([food1]))
+
+        # Should return 1 if food item can be added. 
         self.profile.addFoodStorage(FREEZER, freezer_name)
-        self.assertIsNotNone(self.profile.getOpenFoodStorage())
-        
+        self.assertEqual(1, self.profile.open(FREEZER, freezer_name)) 
```

Comment:
you actually wouldn't have to open it since you already call `addFoodStorage`

Reply:
Got it!